sequencing-server/src/app.ts

```diff
@@ -234,4 +243,14 @@ app.use((err: any, _: Request, res: Response, next: NextFunction) => {
 
 app.listen(PORT, () => {
   logger.info(`connected to port ${PORT}`);
+  logger.info(`Worker pool initialized:
+              Total workers started: ${piscina.threads.length},
+              Heap Size per Worker: ${getEnv().SEQUENCING_MAX_WORKER_HEAP_MB} MB`);
 });
+
+// Immediately call the background transpiler
+await backgroundTranspiler();
+// Then continue calling it every 5 minutes
+setInterval(async () => {
+  await backgroundTranspiler();
+}, 60 * 5 * 1000);
```

Comment:
Is there ever a case where we don't want the background transpiler at all? Maybe add in a way to disable it?

Reply:
Added an option to turn it off via the environmental variables.